src/select-list.h

```diff
@@ -0,0 +1,34 @@
+/*   string/int GtkListStore for GtkComboBox (display name plus value)
```

Comment:
To me it would feel slightly cleaner to add this (and the corresponding source file) as `gtt-select-list.h`. But feel free to leave it as it is.

Reply:
That makes sense.